content/javascript/concepts/dates/terms/getMinutes/getMinutes.md

```diff
@@ -37,3 +37,16 @@ if (today.getMinutes() >= 15) {
 
 // Output: Hurry up, it has been 15 minutes!
 ```
+
+## Codebyte Example
+
+The example below defines a new `Date` object `dateTimeNow`, uses the `getMinutes()` method to extract and store the minutes in a new variable `minutes`, and then logs the current date\time, minutes, and the minutes remaining in the hour.
```

Comment:
```suggestion
The example below defines a new `Date` object `dateTimeNow`, and stores the result of the `getMinutes()` method in a new variable `minutes`. Finally, the current date\time, minutes, and the minutes remaining in the hour are all logged to the console.
```

Reply:
The descriptions for the code examples should be very concise. And explaining all the steps in one sentence can make the sentence too long and difficult to comprehend.